src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_003_02_Export.scala

```diff
@@ -69,17 +70,17 @@ class Pain008_003_02_Export extends LazyLogging {
     NamespaceBinding(null, "urn:iso:std:iso:20022:tech:xsd:pain.008.003.02", nsb3)
   }
 
-  private def getGroupHeaderSDD(rechnungen: List[Rechnung], kontoDatenProjekt: KontoDaten, nbTransactions: String): GroupHeaderSDD = {
-    val MsgId = kontoDatenProjekt.iban.get.slice(0, 15) + getSimpleDateTimeString(getDateTime())
-    val CreDtTm = getDateTime
-    val NbOfTxs = nbTransactions
-    val CtrlSum = None
-    val partyIdentificationSepa1 = kontoDatenProjekt.creditorIdentifier.getOrElse("Initiator")
+  //private def getGroupHeaderSDD(rechnungen: List[Rechnung], kontoDatenProjekt: KontoDaten, nbTransactions: String): GroupHeaderSDD = {
+  //val MsgId = kontoDatenProjekt.iban.get.slice(0, 15) + getSimpleDateTimeString(getDateTime())
+  //val CreDtTm = getDateTime
+  //jval NbOfTxs = nbTransactions
+  //jval CtrlSum = None
+  //jval partyIdentificationSepa1 = kontoDatenProjekt.creditorIdentifier.getOrElse("Initiator")
 
-    GroupHeaderSDD(MsgId, CreDtTm, NbOfTxs, CtrlSum, PartyIdentificationSEPA1(Some(partyIdentificationSepa1), None))
-  }
+  //jGroupHeaderSDD(MsgId, CreDtTm, NbOfTxs, CtrlSum, PartyIdentificationSEPA1(Some(partyIdentificationSepa1), None))
+  //}
 
-  private def getPaymentInstructionInformationSDD(rechnung: Rechnung, kontoDatenKunde: KontoDaten, kontoDatenProjekt: KontoDaten, NbOfTxs: String): PaymentInstructionInformationSDD = {
+  /*private def getPaymentInstructionInformationSDD(rechnung: Rechnung, kontoDatenKunde: KontoDaten, kontoDatenProjekt: KontoDaten, NbOfTxs: String): PaymentInstructionInformationSDD = {
```

Comment:
Cleanup if not needed

Reply:
done, the whole file needs to go. This is the old version